attributes/default.rb

```diff
@@ -20,3 +20,12 @@
 default['rundeck_bridge']['options']['partial-search'] = 'true'
 #<> Default username attribute for the node element in the generated resource xml
 default['rundeck_bridge']['options']['username'] = 'rundeck'
+
+
+# Service setup (Using 'poise')
+default['rundeck_bridge']['poise_service']['options'] = {
```

Comment:
```ruby
default['rundeck_bridge']['poise_service']['options']['upstart'] = {}
default['rundeck_bridge']['poise_service']['options']['systemd'] = {}
```
Any difference if we define it like that?
Seems more explicit (no strong feelings either way, just syntax sugar).